dnsaas/settings.py

```diff
@@ -14,17 +14,18 @@
 
 MANAGERS = ADMINS
 
-if TESTING:
-    DATABASES = {
-        'default': {
-            'ENGINE': 'django.db.backends.sqlite3',
-            'NAME': ':memory:',
-            'USER': '',
-            'PASSWORD': '',
-            'HOST': '',
-            'PORT': '',
-        }
+DATABASES = {
+    'default': {
```

Comment:
use 'TEST' entry to make sure that tests are executed in different db: https://github.com/allegro/ralph/blob/ng/src/ralph/settings/base.py#L114